pandas/tests/series/methods/test_replace.py

```diff
@@ -460,3 +460,17 @@ def test_str_replace_regex_default_raises_warning(self, pattern):
         with tm.assert_produces_warning(FutureWarning) as w:
             s.str.replace(pattern, "")
             assert re.match(msg, str(w[0].message))
+
+    @pytest.mark.parametrize(
+        "s, to_replace, value, expected",
+        [
+            (pd.Series([1]), np.array([1.0]), [0], pd.Series([0])),
```

Comment:
I think you can deduplicate these by using the `frame_or_series` fixture in `pandas/conftest.py`. Then just one test can be added in tests/frame (which is where we try to put tests using the `frame_or_series` fixture).